python/README.md

```diff
@@ -0,0 +1,57 @@
+# Accessing the Cantabular extended API with Python
+
+The API is powered by [GraphQL](https://graphql.org/) which allows users to request only the data that's relevant to them.
+
+The user is presented with a [GraphiQL](https://github.com/graphql/graphiql) session when the API is accessed via a web browser. GraphiQL is the GraphQL integrated development environment (IDE).
```

Comment:
I think GraphiQL is just one IDE rather than "the" IDE.

Reply:
OK